server/src/test/java/org/apache/druid/client/cache/MemcachedCacheTest.java

```diff
@@ -224,6 +217,26 @@ public void emit(Event event)
     }
   }
 
+  @Test
+  public void testSslConnection()
+  {
+    final MemcachedCacheConfig config = new MemcachedCacheConfig()
+    {
+      @Override
+      public boolean enableTls()
+      {
+        return true;
+      }
+
+      @Override
+      public String getHosts()
+      {
+        return "localhost:9999";
+      }
+    };
+    MemcachedCache.create(config);
```

Comment:
this doesn't seem to test much other than making sure setting enableTls doesn't error out. Can we at least have the test somehow verify that the client has tls enabled?

Reply:
Updated the test